### DISCUSSION TITLE
Some windows are just black

### DISCUSSION THREAD

**Discussion by Pallaroid** [Ylianst/MeshCentral#6393] [General]
**Describe the bug**
I have the problem that with some applications I only get a black screen. Unfortunately, the rest is still visible. Only the application itself is not visible or is a black window

**To Reproduce**
1. I connect normally (no RDP)

**Expected behavior**
I want to see all window/application. Many windows work but some do not

**Screenshots**
![Desktop-Nummer2-2024-09-19-18-43](https://github.com/user-attachments/assets/7a6ebe0c-9b40-4467-b39a-3a3f3e595ec3)

**Server Software (please complete the following information):**
 - OS: Ubuntu
 - Virtualization: No
 - Network: WAN
 - Version: 1.1.30
 - Node: ?

**Client Device (please complete the following information):**
 - Device: Laptop / Computer
 - OS: Windows 11 64bit Home
 - Network: Lan (Reached the target/remote device by WAN)
 - Browser: ? 

**Remote Device (please complete the following information):**
 - Device: Laptop
 - OS: Windows 11 23H2/22631 Home 
 - Network: WAN
 - Current Core Version Current Core: Dec 9 2022, 2871199060

**Additional context**
-

**Your config.json file**
{
  "$schema": "https://raw.githubusercontent.com/Ylianst/MeshCentral/master/meshcentral-config-schema.json",
  "__comment1__": "This is a simple configuration file, all values and sections that start with underscore (_) are ignored. Edit a section and remove the _ in front of the name. Refer to the user's guide for >",
  "__comment2__": "See node_modules/meshcentral/sample-config-advanced.json for a more advanced example.",
  "settings": {
    "cert": "XXX",
    "WANonly": true,
    "_LANonly": true,
    "_sessionKey": "MyReallySecretPassword1",
    "port": 443,
    "_aliasPort": 443,
    "redirPort": 80,
    "_redirAliasPort": 80
  },
  "domains": {
    "": {
      "title": "XXX Remote Management",
      "footer": "XXX",
      "_minify": true,
      "_newAccounts": true,
      "_userNameIsEmail": true,
      "agentCustomization": {
        "displayName": "XXX Remote Management",
        "description": "XXX Remote Management",
        "companyName": "XXX Remote Management",
        "serviceName": "XXX",
        "fileName": "XXX",
        "image": "agent-logo.png",
        "icon": "agent_icon.ico",
        "installText": "XXX",
        "_foregroundColor": "100,54,105",
        "backgroundColor": "128,128,128"
      }
    }
  },
  "letsencrypt": {
    "__comment__": "Requires NodeJS 8.x or better, Go to https://letsdebug.net/ first before trying Let's Encrypt.",
    "email": "XXX",
    "names": "XXX",
    "_skipChallengeVerification": true,
    "production": true
  },
  "smtp": {
        "host": "XXX",
        "port": XXX,
        "from": "XXX",
        "user": "XXX",
        "pass": "XXX",
        "tls": false
      }
}

**Comment by PathfinderNetworks** [comment]
What application is it that is showing up black?  Some applications utilize security features to prevent viewing from remote access. I'm wondering if that's the case here?  My other guess would be the video driver on the remote device.

**Comment by si458** [comment]
i agree with @PathfinderNetworks 
without knowing what the application is that doesnt work we really cant help
ive tried googling the image in the taskbar but no results
also exam applications as @PathfinderNetworks explained wont work due to security protections

**Comment by Pallaroid** [comment]
The application is star Money. It is a banking software for managing multiple bank accounts. Is it then possible to view these via RDP? I cant Connect by RDP because Windows 11 Home.

**Comment by si458** [comment]
@Pallaroid oh right yes the chances are they have security in place that detects things like remote control software,
and if present they black screen there application.
sadly i dont think you can enable rdp with windows 11 home sadly apart from upgarding to pro
but win 11 pro oem keys are cheap to buy if you know where to look!

**Comment by si458** [comment]
the only other thing you can try is customise the meshagent like change its name, company, image, etc...
and then uninstall and redownload the new customised version and install it
then see if its still a black screen

https://ylianst.github.io/MeshCentral/meshcentral/customization/#agent-branding
use `agentCustomization` to customise its name and location and service name
then use `agentFileInfo` to customier the data about the file like company name, copyright, icon, etc...
